front/app/modules/commercial/content_builder/admin/containers/index.tsx

```diff
@@ -12,21 +12,27 @@ import { Box } from '@citizenlab/cl2-component-library';
 // craft
 import Editor from '../components/Editor';
 import ContentBuilderToolbox from '../components/ContentBuilderToolbox';
-import ContentBuilderSettings from '../components/ContentBuilderSettings';
 import ContentBuilderTopBar from '../components/ContentBuilderTopBar';
 import ContentBuilderFrame from '../components/ContentBuilderFrame';
+import ContentBuilderSettings from '../components/ContentBuilderSettings';
 
 const StyledRightColumn = styled(RightColumn)`
   min-height: calc(100vh - ${2 * stylingConsts.menuHeight}px);
+  z-index: 2;
+  margin: 0 !important;
```

Comment:
Not a huge fan of this important attributes - are they really necessary? Normally the styled component would override the styles with sufficient specificity.